doc/release-notes.md

```diff
@@ -4,6 +4,20 @@ release-notes at release time)
 Notable changes
 ===============
 
+DoS Mitigation: Mempool Size Limit and Random Drop
+--------------------------------------------------
+This release adds a mechanism for preventing nodes from running out of memory
+in the situation where an attacker is trying to overwhelm the network with
+transactions. This is achieved by keeping track of and limiting the total
+`cost`, a function of a transaction's size in bytes and its fee, of the
+mempool. The maximum total cost is configurable via the parameter
+`mempooltotalcostlimit` which defaults to 80,000,000 (up to 20,000 txs). If a
+node's total mempool `cost` exceeds this limit the node will evict a random
+transaction, weighted by its cost. To prevent a node from re-accepting evicted
```

Comment:
This description is no longer accurate because of the separation of cost and eviction weight. The cost is not dependent on the fee; the eviction weight is.

Reply:
Also, the parameter names do not match the draft ZIP. Either the ZIP or the code (and this description), or both, has to change. Personally I like the `.` after `mempool` to separate it into a different namespace. snake_case vs mushedlowercase is a matter of taste.

Reply:
How about changing this to say ... `a random transaction,`**`preferentially picking larger transactions and ones with below the standard fee.`**`To prevent`...?

Reply:
Note that "and its fee" also needs to be deleted from the definition of cost.

Reply:
I changed the wording a bit here for clarification.